src/objective-c/!ProtoCompiler-gRPCPlugin.podspec

```diff
@@ -37,7 +37,7 @@ Pod::Spec.new do |s|
   # before them.
   s.name     = '!ProtoCompiler-gRPCPlugin'
   v = '1.0.0-pre1'
-  s.version  = v
+  s.version  = "#{v}.2" # .2 to depend on protoc 3.0.0
```

Comment:
It should usually be `"#{v}"` - I started adding `.x` at the end just to be able and push updated versions to the Cocoapods repo. The intention is to get rid of the `.x` as soon as we have another pre-release, or at GA.